doc/guides/bluetooth/overview.rst

```diff
@@ -0,0 +1,98 @@
+.. _bluetooth-overview:
+
+Overview
+########
+
+.. contents::
+    :local:
+    :depth: 2
+
+Zephyr has had a strong focus on Bluetooth and, in particular, on Bluetooth
```

Comment:
I think this would read better, and clarify whos inception you're talking about:
```
Since its inception, Zephyr has had a strong focus on Bluetooth and, in particular, on Bluetooth
Low Energy (BLE).
```